api/src/s2-api.ts

```diff
@@ -48,23 +48,32 @@ async function getPaper(s2Id: string): Promise<Paper | undefined> {
     console.error("Failed to fetch data from Semantic Scholar API", err);
     return;
   }
+
   if (isS2ApiResponseSuccess(response)) {
-    const data = response.data as S2ApiPaper;
-    const year = parseInt(data.year) || null;
+    const data = response.data as S2ApiPaperResponse;
+    if (data.paper === undefined) {
+      return;
+    }
+    var year;
+    if (data.paper.year.text === undefined) {
+      year = null;
+    } else {
+      year = parseInt(data.paper.year.text)
+    }
     return {
       s2Id,
-      title: data.title,
-      authors: data.authors.map(a => ({
-        id: a.authorId,
+      title: data.paper.title.text,
```

Comment:
Jocelyn, do we always know that the properties will be defined and non-null? If you haven't already, can you check to see if these properties will be defined for a set of different papers?